staging/src/k8s.io/apiserver/pkg/server/options/api_enablement.go

```diff
@@ -43,3 +50,47 @@ func (s *APIEnablementOptions) AddFlags(fs *pflag.FlagSet) {
 		"for the core group) can be used to turn on/off specific resources. api/all and "+
 		"api/legacy are special keys to control all and legacy api versions respectively.")
 }
+
+func (s *APIEnablementOptions) Validate(registries ...*registered.APIRegistrationManager) []error {
```

Comment:
This type as a whole needs a big comment discouraging its use and indicating that it won't be part of the recommended options because you should build small apiservers

Reply:
got it.